code/modules/clothing/spacesuits/void.dm

```diff
@@ -1,14 +1,131 @@
+	icon_state = "void-red"
 	item_state = "void"
+	flags_inv = HIDEMASK|HIDEEARS
 
-/obj/item/clothing/suit/space/nasavoid
-	name = "NASA Voidsuit"
-	icon_state = "void"
+/obj/item/clothing/suit/space/voidsuit
+	name = "NASA Void Suit"
+	icon_state = "void-red"
 	item_state = "void"
-	desc = "A high tech, NASA Centcom branch designed, dark red Space suit. Used for AI satellite maintenance."
-	allowed = list(/obj/item/device/flashlight,/obj/item/weapon/tank,/obj/item/device/multitool)
\ No newline at end of file
+	desc = "A retro Space suit designed by a branch at NASA Centcom. This one is in red."
+	allowed = list(/obj/item/device/flashlight,/obj/item/weapon/tank,/obj/item/device/multitool)
+
+//Colors!!!
+/obj/item/clothing/head/helmet/space/voidsuit/green
+	name = "NASA Void Helmet"
+	desc = "A retro space suit helmet designed by a branch at NASA Centcom. This one is in green."
+	icon_state = "void-green"
+
+/obj/item/clothing/suit/space/voidsuit/green
+	name = "NASA Void Suit"
+	icon_state = "void-green"
+	desc = "A retro Space suit designed by a branch at NASA Centcom. This one is in green."
+
+/obj/item/clothing/head/helmet/space/voidsuit/ntblue
+	name = "NASA Void Helmet"
+	desc = "A retro space suit helmet designed by a branch at NASA Centcom. This one is in NT colors."
+	icon_state = "void-ntblue"
+
+/obj/item/clothing/suit/space/voidsuit/ntblue
+	name = "NASA Void Suit"
+	icon_state = "void-ntblue"
+	desc = "A retro Space suit designed by a branch at NASA Centcom. This one is in NT colors."
+
+/obj/item/clothing/head/helmet/space/voidsuit/purple
+	name = "NASA Void Helmet"
+	desc = "A retro space suit helmet designed by a branch at NASA Centcom. This one is in purple."
+	icon_state = "void-purple"
+
+/obj/item/clothing/suit/space/voidsuit/purple
+	name = "NASA Void Suit"
+	icon_state = "void-purple"
+	desc = "A retro Space suit designed by a branch at NASA Centcom. This one is in purple."
+
+/obj/item/clothing/head/helmet/space/voidsuit/yellow
+	name = "NASA Void Helmet"
+	desc = "A retro space suit helmet designed by a branch at NASA Centcom. This one is in yellow."
+	icon_state = "void-yellow"
+
+/obj/item/clothing/suit/space/voidsuit/yellow
+	name = "NASA Void Suit"
+	icon_state = "void-yellow"
+	desc = "A retro Space suit designed by a branch at NASA Centcom. This one is in yellow."
+
+/obj/item/clothing/head/helmet/space/voidsuit/ltblue
+	name = "NASA Void Helmet"
+	desc = "A retro space suit helmet designed by a branch at NASA Centcom. This one is in light blue."
+	icon_state = "void-light_blue"
+
+/obj/item/clothing/suit/space/voidsuit/ltblue
+	name = "NASA Void Suit"
+	icon_state = "void-light_blue"
+	desc = "A retro Space suit designed by a branch at NASA Centcom. This one is in light blue."
+
+//Captian's Suit, like the other captian's suit, but looks better, at the cost of armor
+
+/obj/item/clothing/head/helmet/space/voidsuit/captian

```

Comment:
`Captain` not `captian` (also applies in a few other places in this PR)

Reply:
This one I fully blame on myself being tired, as well as the others miss-spellings